### PR TITLE
Fix/set state warning on server side

### PR DESCRIPTION
**Pull request by yuliaHope**
fix for Warning: setState(...): Can only update a mounting component. This usually means you called setState() outside componentWillMount() on the server. This is a no-op.
Please check the code for the LoadableComponent component.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -165,28 +168,32 @@ function createLoadableComponent(loadFn, options) {
         return;
       }
 
+      let setStateWithMounCheck = (newState) => {
```
Comment (leandrooriente) [inline src/index.js]: Typo: `setStateWithMountCheck`

Comment (yuliaHope) [reply src/index.js]: @leandrooriente  Fixed.

**Comment by anchpop** [comment]
Why has this not been merged?

**Comment by yuliaHope** [comment]
@tajo

**Comment by lmachens** [comment]
any update with this? same warning here

**Comment by klaaz0r** [comment]
I am getting the same warning, following all the example code this error still occurs.

**Comment by rubossq** [comment]
2019 still the same error with SSR

**Comment by leandrooriente** [comment]
Time to move on guys. This library is not well maintained for a long time already.

**Comment by 9teen90nine** [comment]
Hey, any plans to merge this?
This error blocks us from using react-loadable.

File: src/index.js (hunk 1 above)
Comment (Heyxuxiaoting) [reply src/index.js]: That seems to be wrong on serverside

**Pull request merged**